src/snapshots/type_declarations.txt

```diff
@@ -39,36 +47,56 @@ UpperIdent(11:1-11:9),LowerIdent(11:10-11:11),OpColon(11:12-11:13),UpperIdent(11
     (type_decl (3:1-5:4)
         (header "Map" "a" "b")
         (fn (3:11-3:39)
-            (tag (3:32-3:39)
-                "List"
+            (apply (3:32-3:39)
+                (ty "List")
                 (ty_var (3:37-3:38) "b"))
-            (tag (3:11-3:18)
-                "List"
+            (apply (3:11-3:18)
+                (ty "List")
                 (ty_var (3:16-3:17) "a"))
             (fn (3:21-3:27)
                 (ty_var (3:26-3:27) "b")
                 (ty_var (3:21-3:22) "a"))))
     (type_decl (5:1-7:5)
         (header "Foo")
         (tuple (5:7-5:17)
-            (tag (5:8-5:11) "Bar")
-            (tag (5:13-5:16) "Baz")))
+            (ty "Bar")
+            (ty "Baz")))
     (type_decl (7:1-9:6)
         (header "Some" "a")
         (record (7:10-7:42) "<field>" "<field>"))
     (type_decl (9:1-11:9)
         (header "Maybe" "a")
         (tag_union (9:11-9:26) "TODO tags" "TODO open_anno"))
-    (type_decl (11:1-11:37)
+    (type_decl (11:1-13:7)
         (header "SomeFunc" "a")
         (fn (11:14-11:37)
-            (tag (11:29-11:37)
-                "Maybe"
+            (apply (11:29-11:37)
+                (ty "Maybe")
                 (ty_var (11:35-11:36) "a"))
-            (tag (11:14-11:22)
-                "Maybe"
+            (apply (11:14-11:22)
+                (ty "Maybe")
                 (ty_var (11:20-11:21) "a"))
-            (ty_var (11:24-11:25) "a"))))
+            (ty_var (11:24-11:25) "a")))
+    (type_decl (13:1-15:8)
+        (header "MyType")
+        (ty "U64"))
+    (type_decl (15:1-15:24)
+        (header "MyType2")
+        (mod_ty "Module" ".Thingy")))
 ~~~FORMATTED
-NO CHANGE
+module [Map, Foo, Some, Maybe, SomeFunc, add_one, main!]
+
+Map a b : List(a), (a -> b) -> List(b)
+
+Foo : (Bar, Baz)
+
+Some a : { foo : Ok(a), bar : Something }
+
+Maybe a : [Some(a), None]
+
+SomeFunc a : Maybe(a), a -> Maybe(a)
+
+MyType : U64
+
+MyType2 : ModuleThingy
```

Comment:
This seems like a bug if true?

`MyType2 : Module.Thingy`
formats to
`MyType2 : ModuleThingy`
?

Reply:
Oho! Good catch.
It'll be important to verify that the formatted output parses back to the same AST - I'll put that assert on my TODO list.